client/components/first-view/index.jsx

```diff
@@ -43,7 +43,9 @@ const FirstView = React.createClass( {
 	},
 
 	componentWillUnmount() {
-		this.updateDocumentStylesForHiddenFirstView();
+		process.nextTick( () => {
```

Comment:
I don't feel super strong in this particular case, but in general I think we should favor built-in functions as opposed to shims such as `process.nextTick`. Because if every time we could use a standard built-in function we instead used a slightly-semantically-better shim then Calypso would become bloated (whether increasing build time, download size, or runtime efficiency).

Of course, there are always tradeoffs -- and this particular line isn't going to really tip the balance one way or another.